app/models/game-history.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Module dependencies.
+ */
+var mongoose = require('mongoose'),
+    config = require('../../config/config'),
+    Schema = mongoose.Schema;
+
+/**
+ * Answer Schema
+ */
+var GameHistorySchema = new Schema({
+  gameID: { type: String, required: true },
+  started: { type: Date, default: Date.now},
+  ended: Boolean,
```

Comment:
It is good practice to have a default value for fields that are not required. Hence, think of (if possible), default values.  And then use the object value with `type` attribute